code/game/objects/structures/dropship_ammo.dm

```diff
@@ -472,37 +475,72 @@
 		var/list/coords = impact_coords[i]
 		var/turf/detonation_target = locate(impact.x+coords[1],impact.y+coords[2],impact.z)
 		detonation_target.ceiling_debris_check(2)
-		cell_explosion(detonation_target, 250, 90, adminlog = FALSE)
+		cell_explosion(detonation_target, 450, 120, adminlog = FALSE)
 	qdel(src)
 
 /obj/structure/ship_ammo/cas/rocket/napalm
-	name = "\improper XN-99 'Napalm'"
-	desc = "The XN-99 'Napalm' is an incendiary rocket used to turn specific targeted areas into giant balls of fire for a long time. Moving this will require some sort of lifter."
+	name = "\improper AGM-99 'Napalm'"
+	desc = "The AGM-99 'Napalm' is an incendiary rocket used to turn specific targeted areas into giant balls of fire for quite a long time, it has a smaller outer explosive payload than other AGMs, however. Moving this will require some sort of lifter."
 	icon_state = "napalm"
+	travelling_time = 6 SECONDS
 	ammo_id = "n"
-	point_cost = 250
-	devastating_explosion_range = 2
-	heavy_explosion_range = 3
-	light_explosion_range = 4
-	fire_range = 5
+	point_cost = 285
+	devastating_explosion_range = 0
+	heavy_explosion_range = 0
+	light_explosion_range = 8
+	fire_range = 8
 	prediction_type = CAS_AMMO_INCENDIARY
 	cas_effect = /obj/effect/overlay/blinking_laser/incendiary
 
 /obj/structure/ship_ammo/cas/rocket/napalm/detonate_on(turf/impact, attackdir = NORTH)
-	impact.ceiling_debris_check(3)
+	impact.ceiling_debris_check(2)
 	cell_explosion(impact, 250, 90)
 	flame_radius(fire_range, impact, 60, 30) //cooking for a long time
 	var/datum/effect_system/smoke_spread/phosphorus/warcrime = new
-	warcrime.set_up(fire_range + 1, impact, 7)
+	warcrime.set_up(fire_range + 1, impact, 8)
 	warcrime.start()
 	qdel(src)
 
 
+//unguided rockets
+
+/obj/structure/ship_ammo/cas/unguided_rocket
+	name = "RGA-13A 'Sting' "
+	desc = "old unguided rockets found in the strategic warehouses of the UPP combat airfield. Moving this will require some sort of lifter."
```

Comment:
```suggestion
	desc = "Old unguided rockets found in the strategic warehouses of the UPP combat airfield. Moving this will require some sort of lifter."
```